.github/ISSUE_TEMPLATE/documentation_request.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Documentation Request'
+description: 'Request for documentation to be added/altered'
+labels: ['type: Documentation 📚']
+body:
+  - type: 'markdown'
+    attributes:
+      value: |
+        Thanks for filing a documentation request!
+
+        If you have an idea for a new documentation topic, noticed that
+        something is not properly documented, or feel that something is
+        incorrect with the current documentation, you're in the right place!
+  - type: 'input'
+    id: 'subject'
+    attributes:
+      label: 'Subject'
+      description: 'What is the subject (component, function, topic) of this request?'
+      placeholder: 'RadioGroup'
```

Comment:
Can you change the questions here to be somewhat relevant to the gymkhana project?

Like: Is this issue about some typo or error in an existing doc or a request for a new doc?

If new doc, what kind of doc do you want us to add?

Any other information you want to share with us regarding the doc?

Just some questions from the top of my head, can be modified according to the need.